detox/local-cli/test.js

```diff
@@ -115,9 +115,25 @@ module.exports.builder = {
     alias: 'workers',
     group: 'Execution:',
     describe:
-      '[iOS Only] Specifies number of workers the test runner should spawn, requires a test runner with parallel execution support (Detox CLI currently supports Jest)',
-    default: 1,
-    number: true
+      '[iOS Only] Specifies the number of workers the test runner should spawn, requires a test runner with parallel execution support (Detox CLI currently supports Jest)',
```

Comment:
@d4vidi , I wonder if we should remove "[iOS only]" now?

Reply:
Yikes... yes, you may :)